bodhi/client/bindings.py

```diff
@@ -855,6 +856,66 @@ def get_releases(self, **kwargs):
         """
         return self.send_request('releases/', verb='GET', params=kwargs)
 
+    @errorhandled
+    def save_release(self, **kwargs):
+        """
+        Save a new or edited release.
+
+        Args:
+            kwargs (dict): The parameters to send with the request.
+        """
+        res = self.send_request('releases/', verb='POST', auth=True,
+                                data=kwargs)
+
+        if 'errors' in res:
+            self.print_errors(res)
+
+        else:
+            print("Saved release:")
```

Comment:
The bindings should not assume they have stdout/stderr because they are intended for general use (and Bodhi server uses them too).

Reply:
Note also that we should use ```click.echo()``` instead of ```print()```.